src/helpers/_mixins.scss

```diff
@@ -2,9 +2,6 @@
  * _mixins.scss
  */
 
-/*
- * General mixins
- */
 @mixin clearfix { overflow: hidden; }
```

Comment:
IMO we should get rid of this, or change it to be what `clearfix-overflow-visible` does.

Reply:
I agree, we should remove the clearfixes altogether.